tests/test_script.py

```diff
@@ -66,3 +67,7 @@ def test_pyc():
 def test_version():
     assert call_vulture(['--version']) == 0
     assert get_output(['--version']).strip() == 'vulture ' + __version__
+
+
+def test_repo():
+    assert call_vulture([LINES]) == 0
```

Comment:
Good idea to test this, but let's not test each file separately. I suggest to put the following under the test_script_* functions:

    def test_module():
        assert call_vulture(['vulture']) == 0